index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navmenu">
+        <h3>My Logo</h3>
+        <img src="image/Union.png" alt="" />
+      </nav>
+    </header>
+    <section class="backgroundimg">
+      <div class="menuSection">
+        <h1 class="introduction">I’m Mariane Glad to see you!</h1>
+        <p class="Whoiam">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+      </div>
+      <div class="menuSection">
+        <h3 class="connect">LET’S CONNECT</h3>
```

Comment:
- Please make this a link and style it appropriately as it will be used to link to your contact section in the subsequent phase of this project.  like `
```
<h3>< a href ="#" > Let's Connect </a></h3> 
```